.github/workflows/ci.yml

```diff
@@ -36,5 +36,11 @@ jobs:
             cat errors.csv
             exit 1
           fi
+        - name: Validate html including missing uuid
+          run: |
+            validate-mbz-html html errors.csv html --uuids-populated
+            CSV_LINES=$(wc -l < errors.csv )
+            echo "Validation errors including missing uuid errors found:"
+            cat errors.csv
```

Comment:
I think this is indented too far leading to a [syntax error in the CI](https://github.com/openstax/k12-contents-raise/actions/runs/6795213052). (I expect it will still error / fail until you merge your `raise-mbtools` PR even once this is fixed)